INSTALL.md

```diff
@@ -12,14 +12,15 @@ To build the software do:
 ./gradlew build
 ```
 
-## Releasing a new version
+## Releasing and publishing a new version
 
-We use the [Axion Release Plugin](https://github.com/allegro/axion-release-plugin) to release new versions.
+We use the [Axion Release Plugin](https://github.com/allegro/axion-release-plugin) to release new versions and the [Gradle Nexus Publish Plugin](https://github.com/gradle-nexus/publish-plugin).
+to publish new versions to the Open Source Sonatype Nexus repository (and therefore to Maven Central).
```

Comment:
Full stop and start a new sentence with a capital letter or continue with the sentence?